module-check/src/main/clojure/clojure/core/typed/base_env.clj

```diff
@@ -1449,6 +1449,11 @@ clojure.repl/pst (IFn [-> nil]
                       [(U Int Throwable) -> nil]
                       [Throwable Int -> nil])
   )
+
```

Comment:
Does this actually compile? Annotations should be in the var-mappings expression.